services/s3-service.js

```diff
@@ -0,0 +1,35 @@
+const AWS = require('aws-sdk');
+class S3Service {
+
+    constructor() {
+        this.s3 = new AWS.S3();
+    }
+
+    async createPreSignedURL(bucketName, prefix, fileName) {
+        try {
+            const params = {
+                Bucket: bucketName,
+                Key: `${prefix}`,
```

Comment:
I think this should be **`${prefix}/${fileName}`**

Reply:
updated the object key